metrics/prometheus.go

```diff
@@ -186,6 +186,38 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 					}
 					return values
 				},
+			}, {
+				name:        "container_cpu_user_seconds",
```

Comment:
Since this is cumulative, it should have the _total suffix

Reply:
But this will induce conflict with
```
name:      "container_cpu_user_seconds_total",
help:      "Cumulative user cpu time consumed in seconds.",
```
although i can change word user to usermode if necessary

Reply:
Right.  See my comment above.  This change should add a dimension to existing metrics, rather than create new ones.